windows_containers/creating_windows_machinesets/creating-windows-machineset-gcp.adoc

```diff
@@ -0,0 +1,24 @@
+:_content-type: ASSEMBLY
+[id="creating-windows-machineset-gcp"]
+= Creating a Windows `MachineSet` object on GCP
+include::_attributes/common-attributes.adoc[]
+:context: creating-windows-machineset-gcp
+
+toc::[]
+
+You can create a Windows `MachineSet` object to serve a specific purpose in your {product-title} cluster on Google Cloud Platform (GCP). For example, you might create infrastructure Windows machine sets and related machines so that you can move supporting Windows workloads to the new Windows machines.
+
+[discrete]
+== Prerequisites
+
+* You installed the Windows Machine Config Operator (WMCO) using Operator Lifecycle Manager (OLM).
+* You are using a supported Windows Server as the operating system image. 
+
+include::modules/machine-api-overview.adoc[leveloffset=+1]
+include::modules/windows-machineset-gcp.adoc[leveloffset=+1]
+include::modules/machineset-creating.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+== Additional resources
```

Comment:
This [needs an ID](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#anchoring-prerequisites-additional-resources-and-next-steps-titles-in-assemblies), something like:
```suggestion
[id="additional-resources_creating-windows-machineset-gcp"]
== Additional resources
```